### PR TITLE
docs: 添加了 chapter 2

### PR DESCRIPTION
**Pull request by echoLC**

### CODE REVIEW

File: manual/chapter2/getHTML.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Get HTML Content
+`editor.txt.html()` API be used to get HTML content.
+
+Note **the html which get from editor doesn't contains any style**.it means that you can implement changing theme function by yourself.if you want to custom the styles for tags such as `<table>`, `<code>`, `<blockquote>` and so on, The styles used in the editor are provided below for your reference：
```
Comment (hahaaha) [inline manual/chapter2/getHTML.md:4]: 这一行的最后一行The styles used in the editor are provided below for your reference: 中的the style used in the editor 换成 the editor styles 会不会更好一点

File: manual/chapter2/getJSON.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "tag": "p",
+        "attrs": [],
+        "children": [
+            "欢迎使用 ",
+            {
+                "tag": "b",
+                "attrs": [],
+                "children": [ "wangEditor" ]
+            },
+            " 富文本编辑器"
```
Comment (WangJolin) [inline manual/chapter2/getJSON.md:18]: 这里的中文可以改成英文

File: manual/chapter2/getJSON.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                    { "name": "style", "value": "max-width:100%;" }
+                ]
+            }
+        ]
+    }
+]
+```
+
+## Set JSON
+You aslo can use `editor.txt.setJSON(json)` API to set JSON content, the json param format must be consistent with mentioned above.
```
Comment (WangJolin) [inline manual/chapter2/getJSON.md:38]: You can also

File: manual/chapter2/getHTML.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Get HTML Content
+`editor.txt.html()` API be used to get HTML content.
```
Comment (WangJolin) [inline manual/chapter2/getHTML.md:2]: The xxx API is used to ?

File: manual/chapter2/getJSON.md (hunk 3 above)
Comment (WangJolin) [reply manual/chapter2/getJSON.md:38]: “欢迎使用”和“富文本编辑器”可以改成英文的更合适

File: manual/chapter2/getText.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# Get Text Content
+The API `editor.txt.text()` is used to get text content.
```
Comment (WangJolin) [inline manual/chapter2/getText.md:2]: the API of xxx is used to ...会不会比较合适？

File: manual/chapter2/history.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# History Setting
+At present, history has two modes: standard mode(for modern browsers) and compatible mode (for IE and old Edge).You can set history mode by `editor.config.compatibleMode` option.You aslo can set history max size by `editor.config.historyMaxSize` option.
```
Comment (WangJolin) [inline manual/chapter2/history.md:2]: There are two ways to set History：standard mode(for modern browsers) and compatible mode (for IE and old Edge). You can set mode by `editor.config.compatibleMode`. And about the max size of History, you can set it by `editor.config.historyMaxSize`. The following shows how to use them.

File: manual/chapter2/history.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+when you set compatible mode, if you want to modify delay time for recording(when user is not active after x millisecond),
```
Comment (WangJolin) [inline manual/chapter2/history.md:4]: The API of  `editor.config.onchangeTimeout` can be used to set the delay time of history records (when users are not active after x millisecond) on compatible mode.
（仅供参考）

**Review by WangJolin — APPROVED** [review]
评论中的写法仅供参考

File: manual/chapter2/history.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+// by default，IE and old Edge use compatible mode，if you want to set the mode for other browsers, you can set a function.
+editor.config.compatibleMode = function () {
+    // return true for compatible mode, otherwise return false for standard mode.
+    return true
+}
+
+// when we use compatible mode, we can set frequency time for recording，the default value is 200 ms.
+editor.config.onchangeTimeout = 500 // change to 500 ms
+
+// you aslo can set max siz for history, the default value is 30.
```
Comment (hahaaha) [inline manual/chapter2/history.md:20]: set max siz 这里 应该是set max size

File: manual/chapter2/history.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+const E = window.wangEditor
+const editor = new E("#div1")
+
+// by default，IE and old Edge use compatible mode，if you want to set the mode for other browsers, you can set a function.
+editor.config.compatibleMode = function () {
+    // return true for compatible mode, otherwise return false for standard mode.
+    return true
+}
+
+// when we use compatible mode, we can set frequency time for recording，the default value is 200 ms.
```
Comment (hahaaha) [inline manual/chapter2/history.md:17]: we can set frequency time for recording。 这里的 frequency time for recording  用recording time 会更好一点

File: manual/chapter2/setContent.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
+# Set Content
+You should try you best to use first way in follow several ways, because it is the most efficient.
```
Comment (hahaaha) [inline manual/chapter2/setContent.md:2]: if conditions allow, you best to use first way in follow several ways 这样是不是更好一点

**Pull request closed without merging**